blueocean-core-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jenkins-cd/blueocean-core-js",
-  "version": "0.0.171",
+  "version": "1.10.3",
```

Comment:
There are lots of references to `@jenkins-cd/blueocean-core-js@0.0.171` in the various `package.json` files that probably should be updated, too.

Reply:
Should their?

I really do not know since BO switched to core-js being a hpi plugin and auto-overriding those version anyway. 

I asked that changes to the core-js plugin (a js only package) should been published to npm to be consumable as npm upstream but the latest security updates to fetch had not been published, 

As I understand the only ones that need that version to be in sync are upstream consumer so no other package.json in BO is minding that version anyway, saying it, since 11 month ago I needed the version to be changed (git blamed myself) and now (no other changes).

...I actually published that version in latest version as npm package to make it usable in our company project.